tests/hopp/test_hybrid.py

```diff
@@ -1392,7 +1438,7 @@ def reinstate_orig_values():
     )
 
     capcred = hybrid_plant.capacity_credit_percent
-    assert capcred["pv"][0] == approx(8.03, rel=0.05)
+    assert capcred["pv"][0] == approx(6.85, rel=0.05)
```

Comment:
I don't understand why this would decrease, but perhaps I do not understand what it is.

Reply:
This is where the capacity credit percent is calculated, it looks to me that it should increase given increased AEP, so it does seem strange. https://github.com/NREL/HOPP/blob/53490cb54abfe024e6338310a88374707d54ed20/hopp/simulation/technologies/power_source.py#L210

Reply:
Thanks for that. I wonder if the problem is in changes made to the `interconnect_kw` for dispatching. I have not investigated, just throwing that out for now.